### ISSUE TITLE
[bug] customDevices.json hot reload only works once

### ISSUE THREAD

**Issue by pdbogen** [OpenZWave/Zwave2Mqtt#670]
**Version** 4.0.0

Build/Run method
- [x] Docker
- [ ] PKG
- [ ] Manually built (git clone - npm install - npm run build ) 

Zwave2Mqtt version: 4.0.0
Openzwave Version: 1.6.

**Describe the bug**

vim (and probably other editors) update files by `rename`ing a new file on top of the old file. NodeJS's `fs.watch` watches based on inode, and so only watches the deposed "old file", after a single event that registers the renaming.

Instead, when `fs.watch` handler receives an eventtype `rename`, it needs to close the old watch and re-create the watch to watch the new file.

There's a potential race condition here wherein the file doesn't exist when it wants to be read, and coördinating with the desired "read at most once per sceond" behavior adds further complexity.

**To Reproduce**

start zwave2mqtt with a `customDevices.json` on disk. edit it with `vim`. observe that after the first time saving the file, zwave2mqtt no longer detects changes.

here's a minimal reproducer of the issue:

```nodejs
process.on('SIGINT', () => { console.log("exiting..."); process.exit(); })

const fs = require('fs')

const watch = (file) => {
  var lock = false
  fs.watch(file, () => {
    if(lock) { return }
    lock = true

    console.log("here, would read " + file)

    setTimeout(() => { lock = false }, 1000)
  })
}

console.log("waiting...")
watch("/work/test")
```

Run this like: `touch test && docker run --rm -it -v $PWD:/work node node /work/node.js`

Then play around with the `test` file in the same directory, edit it with vim, etc.

**Expected behavior**
zwave2mqtt should reload the file any time it's changed, regardless of whether it's deleted/re-created/renamed/etc. In fact, zwave2mqtt should load the file the first time it's created, even if it didn't exist before.

**Additional context**

**Comment by robertsLando** [comment]
@pdbogen Do you also have a suggested solution to this?

**Comment by robertsLando** [comment]
Fixed in 4.0.2
